onnxruntime/core/providers/cuda/cuda_provider_interface.cc

```diff
@@ -0,0 +1,14 @@
+#include "core/session/onnxruntime_c_api.h"
```

Comment:
missed this: Please include Microsoft copyright notice and header here.

Reply:
@guoyuhong, please confirm it in https://github.com/microsoft/onnxruntime/pull/16437.

Reply:
I will change this in later PR.

Reply:
@tianleiwu has added the head in https://github.com/microsoft/onnxruntime/pull/16437 ;-)